src/main/java/cromwell/parser/RuntimeKey.java

```diff
@@ -13,8 +13,8 @@
     DEFAULT_ZONES("defaultZones", JES),
     DOCKER("docker", new BackendType[]{JES}, LOCAL), // Alternate constructor due to both optional and mandatory backends
     FAIL_ON_STDERR("failOnStderr", JES, LOCAL, SGE),
-    FAIL_ON_RC("failOnRc", LOCAL, SGE),
     MEMORY("memory", JES),
+    NO_FAIL_ON_RC("noFailOnRc", LOCAL, SGE),
```

Comment:
I'd love to have a term for this that didn't include a negation, doing some of the boolean logic below in my head gets painful.  :smile:  `CONTINUE_ON_RETURN_CODE` or something like that?